docs/user/interactive_customizing.rst

```diff
@@ -14,55 +14,118 @@ Important files
 
 The launch is enabled by the content in the following files in your project:
 
-* language-specific files like ``requirements.txt`` or ``install.R``
-
 * ``Dockerfile``: defines the type of interactive environment and other software
   installed in the environment, including the ``renku`` command-line installation.
 
 * ``.gitlab-ci.yml``: controls the docker build of the image based on the project's
   ``Dockerfile``.
 
+* ``requirements.txt`` or ``install.R``: language-specific files controlling the
+  libraries.
+
+* ``.renku/renku.ini``: renku project configurations containing a
+  ``[renku "interactive"]`` section.
+
 The most basic modifications are installations of additional packages. This can be
 done automatically for Python and R projects if you add the packages you want
 to ``requirements.txt`` and ``install.R`` respectively.
 
 
+Renku project configurations
+----------------------------
+
+When starting a new Interactive Environment, most of the options can be manually
+changed by the user. Depending on the specific RenkuLab deployment, you can select
+more RAM, a higher CPU quota, etc.
+
+Your project may even include a package with an advanced UI (like
+`Streamlit <https://renku.discourse.group/t/how-to-deploy-streamlit-in-renku/169>`_)
+and you probably want to choose it as default.
+
+It's possible to set a default value for all these options using the project
+configurations stored in the ``.renku/renku.ini`` file.
+Once you do that, each time a user tries to start a new environment, those options will
+be pre-selected.
+
+.. note::
+
+  Manually modifying the ``renku.ini`` file is not recommended.
+  You can use the
+  `renku config command <https://renku-python.readthedocs.io/en/latest/commands.html#module-renku.cli.config>`_
+  from an interactive environment, for example:
+
+    renku config set interactive.default_url "/tree"
+
+  We are working on adding a user-friendly solution to set default options on
+  the project's settings page.
+
+**What are the specific options?**
+
+You can find a comprehensive list of options :ref:`on this page <renku_ini>`. Most commonly,
+you may want to change the ``default_url`` or set a specific ``image``.
+
+The first case is useful when you prefer to show a different default UI, like the standard
+Jupyter interface ``/tree``, or when you need support for a different interface,
+like R studio ``/rstudio`` or  ``/streamlit`` (not included in the standard Python template).
+
+The ``image`` is useful when you settle on a Docker image and you don't need to change it
+anymore. The benefit is particularly evident when building a new image takes a lot of time
+(e.g. you added big packages) or when you expect the project to be used by a lot of people
+over a short period of time (e.g. you use it in a presentation or a lecture).
+
+Even if it's common to start the environment with the default values, setting a default value
+doesn't prevent a user from changing it.
```

Comment:
I do not think a user can change the pinned image, or am I wrong?

Reply:
This is true in the current UI implementation, but I'm not sure if we want to change that or not.
The section is about all the options, but you are right, `image` is an exception. If you think it's better to specify it, I can add the text

Reply:
I think we should describe the current UI behavior in the documentation. If that changes, then we should update the documentation.